packages/editor-ui/src/components/NDVDraggablePanels.vue

```diff
@@ -105,6 +108,9 @@ export default Vue.extend({
 
 		window.addEventListener('resize', this.setTotalWidth);
 		this.$emit('init', { position: this.mainPanelDimensions.relativeLeft });
+		setTimeout(() => {
```

Comment:
I think it's generally nicer to use `$nextTick` for this kind of operations since it's more aware of the life-cycle of the component and DOM update

Reply:
I have experienced some differences between setTimeout and `$nextTick` and have found `setTimeout` to be reliable..  unless there's a good reason would rather stick to setTimeout